prac_05/state_names.py

```diff
@@ -0,0 +1,49 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+CODE_TO_NAME = {"QLD": "Queensland",
+                "NSW": "New South Wales",
+                "NT": "Northern Territory",
+                "WA": "Western Australia",
+                "ACT": "Australian Capital Territory",
+                "VIC": "Victoria",
+                "TAS": "Tasmania"}
+
+MAX_KEY_LEN = len(max(CODE_TO_NAME, key=len))
+MIN_KEY_LEN = len(min(CODE_TO_NAME, key=len))
+MAX_VALUE_LEN = len(CODE_TO_NAME[max(CODE_TO_NAME, key=len)])
+MIN_VALUE_LEN = len(CODE_TO_NAME[min(CODE_TO_NAME, key=len)])
```

Comment:
When I first saw these constants I was a bit confused and overwhelmed, are all of these constants needed?
We already know what the max length of the keys are... 3.
Also you didn't use either of the MIN constants, redundant code.

Reply:
So with this it's for practice using these types of lines, I also think it is maybe more robust than using an integer(what if we decide to add something that isn't length 3). I guess with the context of the problem 3 is probably more appropriate though. MIN was just more practice and just for future use again. Probably inappropriate for the problem as set.